web/src/routes/LicensePage.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { License, parseLicenses } from 'api-client/src/utils/licences'
+
+import Caption from '../components/Caption'
+import List from '../components/List'
+import ListItem from '../components/ListItem'
+import { reportError } from '../utils/sentry'
+
+const LicenseContainer = styled.div`
+  padding: 5px 15px;
+  line-height: 120%;
+`
+
+type PropsType = {
+  name: string
+  version: string | undefined
+  license: string
+  onPress: string
+}
+
+const LicenseItem = (props: PropsType): ReactElement => {
+  const { name, version, license, onPress } = props
+  return (
+    <ListItem path={onPress} title={name}>
+      <LicenseContainer>
+        {`version: ${version}`}
+        <br />
+        {`license: ${license}`}
+      </LicenseContainer>
+    </ListItem>
+  )
+}
```

Comment:
Should be moved to its own file or just inlined below.

Reply:
Imo it is not necessary to move 20 loc in its own file when it's only used in this file

Reply:
@f1sh1918 @SteffiStuffel what do you think? Do we have/should we have a policy for this?

Reply:
I just extracted it into its own file